packages/swing-store/src/swingStore.js

```diff
@@ -194,6 +194,21 @@ export function makeSwingStore(dirPath, forceReset, options = {}) {
     // { verbose: console.log },
   );
 
+  function setUnsafeFastMode(enabled) {
+    const journalMode = enabled ? 'off' : 'wal';
+    const synchronousMode = enabled ? 'normal' : 'full';
+    !db.inTransaction || Fail`must not be in a transaction`;
+
+    db.unsafeMode(!!enabled);
+    const actualMode = db.pragma(`journal_mode=${journalMode}`, {
```

Comment:
Note that WAL mode cannot be turned off once it's been turned on.. this `actualMode` check is awesome, but let's add a note here to explain that it might fail because WAL mode is permanent.